src/machine/board_hw-651.go

```diff
@@ -0,0 +1,72 @@
+//go:build hw_651
+
+package machine
```

Comment:
I'd like some description here about this board, who makes it, best if link to official data sheet / documentation.
See other board files.